page-searchbox_multi.php

```diff
@@ -0,0 +1,133 @@
+                      >Full view only</label
+                    >
+                  </div>
+                </div>
+              <div>
+                <label class="offscreen" for="searchfilter">Filter by Library</label>
+                <select style="width:100%" id="searchfilter" name="filter[]" >
+                  <option value="">All Libraries</option>
+                  <option value="htsource:Boston College">Boston College</option>
+                  <option value="htsource:Columbia University">Columbia University</option>
+                  <option value="htsource:Cornell University">Cornell University</option>
+                  <option value="htsource:Duke University">Duke University</option>
+                  <option value="htsource:Harvard University">Harvard University</option>
+                  <option value="htsource:Indiana University">Indiana University</option>
+                  <option value="htsource:Library of Congress">Library of Congress</option>
+                  <option value="htsource:Minnesota Digital Library">Minnesota Digital Library</option>
+                  <option value="htsource:New York Public Library">New York Public Library</option>
+                  <option value="htsource:North Carolina State University">North Carolina State University</option>
+                  <option value="htsource:Northwestern University">Northwestern University</option>
+                  <option value="htsource:Penn State University">Penn State University</option>
+                  <option value="htsource:Princeton University">Princeton University</option>
+                  <option value="htsource:Purdue University">Purdue University</option>
+                  <option value="htsource:Universidad Complutense de Madrid">Universidad Complutense de Madrid</option>
+                  <option value="htsource:University of California">University of California</option>
+                  <option value="htsource:University of Chicago">University of Chicago</option>
+                  <option value="htsource:University of Florida">University of Florida</option>
+                  <option value="htsource:University of Illinois">University of Illinois</option>
+                  <option value="htsource:University of Michigan">University of Michigan</option>
+                  <option value="htsource:University of Minnesota">University of Minnesota</option>
+                  <option value="htsource:University of North Carolina">University of North Carolina</option>
+                  <option value="htsource:University of Virginia">University of Virginia</option>
+                  <option value="htsource:University of Wisconsin">University of Wisconsin</option>
+                  <option value="htsource:Utah State University Press">Utah State University Press</option>
+                  <option value="htsource:Yale University">Yale University</option>
+                </select>
+              </div>
+            </form>

```

Comment:
I can see that github is really bent out of shape about something here, but I ran this through an HTML validator and (outside of `<main>` not being a valid XMTL 1.0 element), everything is fine. So 🤷‍♀️